### PR TITLE
Impove testing and add php 7.1 and 7.2 support

### PR DESCRIPTION
**Pull request by snapshotpl**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,59 +2,60 @@ sudo: false
...
-    - zf-mkdoc-theme
 
 env:
   global:
-    - SITE_URL: https://zendframework.github.io/zend-stdlib
-    - GH_USER_NAME: "Matthew Weier O'Phinney"
-    - GH_USER_EMAIL: matthew@weierophinney.net
-    - GH_REF: github.com/zendframework/zend-stdlib.git
-    - secure: "l+YU9Igd9IUA60PE+iC4AZt2hHY9hpsSB7S2jck9fjPm0+15UCGk2G1qg5hCi+FlPslXhk6wOLLDPc40BiCrwH+m5CP9cIdkGTuNc9rVF9S5IiRLkOVxeTcdE+JRm7hZ2NBgxEiTuBLs5RlN39o41zPDFllgnJv69UmmUIMHuG3NwJJiRE9NA2EMIhZwx3UT/Zqq94MzAbYXi2xspoQ53Z/rfvVSJK6tdARp31q4NWvlYsc2YVcEWv+j3VY5luMSCMVQVNBIuXdd0hYpGt2o3H6S9azzcKRQB7saoAT/tiAEj5121tv3j3cKNQQMzI4xv7RrtQEiAhumeU3AsVDyqBmqqUQNwpnWH0RZdRry4jUKJgx0TK0yZH8ewdz6DhktBqS+c2a/FzfO3yc+LeuIy3i73NezNwCgwviOBYbWPAo4LqKtJ7gdTbVVdJaUeCtKTZ1b1k4MtaxjZjz0IK5+rxJsZ1DQF5fhJIzkjXqEEON2mHQu/qE+I7ow4Jvfs0T+KYJW6Yf5qFzBkrTUNjV60rmawaEjgxgMvtwN+MvcQ/X2OWkdjdzqzQIDdjssdRu9IBPU/G2uqzS8POL9o1j8H3NE430Tsbg+jiENlqmU+ra5wuRrmTBToli1kpgl9LyhgvjGokxAB+mwwKv7FiIQtBmSgEvsD112eSYl3HVWpdE="
+    - COMPOSER_ARGS="--no-interaction --no-plugins"
```
Comment (webimpress) [inline .travis.yml]: I think we don't need `--no-plugins` flag here.

File: composer.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     },
     "require-dev": {
         "athletic/athletic": "~0.1",
-        "phpunit/PHPUnit": "~4.0",
+        "phpunit/phpunit": "^5.7.25",
```
Comment (webimpress) [inline composer.json]: Hm... not so easy, need more work to update PHPUnit ...

Comment (webimpress) [reply composer.json]: Wow, actually I'm so surprised, tests pass ! We must have there only basic tests then:)

Comment (snapshotpl) [reply composer.json]: :) One thing to do (in next pr) it's to add support for phpunit 6.

File: .travis.yml (hunk 1 above)
Comment (snapshotpl) [reply .travis.yml]: it's default flag in many zf repos

Comment (webimpress) [reply .travis.yml]: Hm... I don't know where you can see it. Definitely we don't have it in template in the maintainers repo: https://github.com/zendframework/maintainers/blob/master/template/.travis.yml#L11

**Comment by webimpress** [comment]
@snapshotpl I just noted I did similar changes before to composer.json and Travis CI config, see for example commits: 
- https://github.com/zendframework/zend-stdlib/pull/85/commits/6e9335b074a88a4f68b3fc37a92efd3c77f4097b
- https://github.com/zendframework/zend-stdlib/commit/65a123c61f3e41c1efda52c3956e6d00c02b0e16

and some of them are already on develop branch, see: https://github.com/zendframework/zend-stdlib/compare/develop

**Comment by weierophinney** [comment]
Thanks, @snapshotpl

**Comment by weierophinney** [comment]
Merged to develop.

**Pull request closed without merging**